bpmn-model/src/main/java/io/zeebe/model/bpmn/validation/zeebe/ProcessValidator.java

```diff
@@ -33,8 +36,38 @@ public void validate(Process element, ValidationResultCollector validationResult
 
     final Collection<StartEvent> topLevelStartEvents =
         element.getChildElementsByType(StartEvent.class);
-    if (topLevelStartEvents.size() != 1) {
-      validationResultCollector.addError(0, "Must have exactly one start event");
+    if (topLevelStartEvents.size() == 0) {
+      validationResultCollector.addError(0, "Must have atleast one start event");
+    } else {
+      final Collection<StartEvent> noneStartEvents =
+          topLevelStartEvents
+              .stream()
+              .filter(startEvent -> isTimerEvent(startEvent) || isNoneEvent(startEvent))
+              .collect(Collectors.toList());
+      if (noneStartEvents.size() > 1) {
+        validationResultCollector.addError(
+            0, "Must have exactly one none start event or one timer start event");
+      }
+      if (noneStartEvents.size() == 1 && topLevelStartEvents.size() > 1) {
+        validationResultCollector.addError(0, "Cannot have start events of different types");
+      }
     }
   }
+
+  private boolean isNoneEvent(StartEvent startEvent) {
+    if (startEvent.getEventDefinitions().size() == 0) {
```

Comment:
Since you're returning a boolean, you can just:

```suggestion
    return startEvent.getEventDefinitions().size() == 0;
```